.zapierapprc

```diff
@@ -0,0 +1,4 @@
+{
+  "id": 166479,
+  "key": "App166479"
```

Comment:
Nothing wrong here - just want to point out that Fer ran into a similar problem with Environmental variables when he was missing an [includeInBuild](https://gitlab.com/zapier/team-integrations/zapier-platform-integrations/-/blob/main/apps/google-forms/.zapierapprc#L5) property we usually have in our monorepo

Reference:
https://platform.zapier.com/cli_docs/cli#build

Error:
https://cdn.zappy.app/358b7d39432b533bda66494a265f46da.png

Reply:
Ty, when is this error showing up?

Reply:
I think it showed trying to authenticate in the zapier editor. Maybe not an issue with this app you made but it's worth mentioning we do have that extra property to include additional files needed for the build.